www/src/utils/presets.js

```diff
@@ -9,8 +9,8 @@ module.exports = {
     sm: `@media (min-width: 550px)`,
     Md: `(min-width: 750px)`,
     md: `@media (min-width: 750px)`,
-    Lg: `(min-width: 1000px)`,
-    lg: `@media (min-width: 1000px)`,
+    Lg: `(min-width: 1050px)`,
```

Comment:
```suggestion
    Lg: `(min-width: 1000px)`,
```

Can we avoid changing the breakpoint?

Reply:
You can, but it causes a horizontal scroll-bar.

Reply:
I agree with @sidharthachatterjee — we shouldn't adjust the breakpoint constant, but instead opt-out of using it to define when the mobile navigation is hidden/shown, and instead use a bespoke media query (`@media (min-width: [whatever value we actually need]`).

Reply:
On another note, I worked on #12603 over the weekend, which (among other stuff) changes how the main navigation items look, which gains us a bit of space:

![image](https://user-images.githubusercontent.com/21834/54544750-8bc43f00-49a0-11e9-9b6d-1761e66ddf6b.png)

@sidharthachatterjee #12603 is ready to go — maybe we merge that, and then check how things look here with the "Starters" item added? Happy to merge `master` here.

Reply:
Dropping the uppercase does seem to fix the spacing issue even with the breakpoint reset to `1000px` and leaving starters in place.